stencil-library/src/components/payment-form/payment-form.stories.ts

```diff
@@ -148,3 +151,19 @@ Styled.args = {
   --jfi-radio-button-group-width: 100%;
   `,
 };
+
+export const Slotted = Template.bind({});
```

Comment:
I don't think we should add a story until we actually have a insurance component to show here